src/output-json-ftp.c

```diff
@@ -99,7 +99,7 @@ static json_t *JsonFTPLogCommand(Flow *f, FTPTransaction *tx)
         TAILQ_FOREACH(response, &tx->response_list, next) {
             /* handle multiple lines within the response, \r\n delimited */
             uint8_t *where = response->str;
-            uint16_t length = response->len;
+            uint16_t length = (uint16_t) strlen((const char *)response->str);
```

Comment:
Why is `len` not the same as `strlen`? Or is that another off by one because of the null-byte?

Reply:
Here, `request->len` includes the terminating NULL.